src/color/background.json

```diff
@@ -1,4 +1,280 @@
 {
+  "background": {
```

Comment:
It is part of the core Style Dictionary API that when we use colors that we need to start the JSON with a `color` object. Without this we lost core capabilities to transform values as needed.

Reply:
@blackfalcon this seems like it maybe needs to be a further discussion? I am not really sure who named the tokens, but there was definitely a conscious decision to leave the `color` part out of the naming convention.